### PR TITLE
 Add example for Intl.listFormat.prototype.resolvedOptions()

### PR DESCRIPTION
**Pull request by IestynGage**
First time contributing to this project 😄 

#1648

Edit: Oops included the wrong number 🤦‍♂️

### CODE REVIEW

File: live-examples/js-examples/intl/meta.json [hunk 1]
```diff
@@ -78,6 +78,12 @@
             "title": "JavaScript Demo: Intl.GetCanonicalLocales",
             "type": "js"
         },
+        "intlListFormatPrototypeResolvedOptions": {
+            "exampleCode": "./live-examples/js-examples/intl/intl-listformat-prototype-resolvedoptions.js",
+            "fileName": "intl-listformat-prototype-resolvedoptions.html",
+            "title": "JavaScript Demo: Intl.ListFormat.prototype.resolvedOptions()",
+            "type": "js"
+        },
```
Comment (wbamberg) [inline live-examples/js-examples/intl/meta.json:86]: this should go after "intlListFormat" (really we should have CI that checks these JSON files are alphabetised).

File: live-examples/js-examples/intl/intl-listformat-prototype-resolvedoptions.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const deListFormatter1 = new Intl.ListFormat('de-DE', { style: 'short', type: 'disjunction' });
```
Comment (wbamberg) [inline live-examples/js-examples/intl/intl-listformat-prototype-resolvedoptions.js:1]:
```suggestion
const deListFormatter1 = new Intl.ListFormat('de-DE', { type: 'disjunction' });
```

maybe, so we can show the defaults too? (of course you'd need to change the expected output as well)

File: live-examples/js-examples/intl/intl-listformat-prototype-resolvedoptions.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+const options1 = deListFormatter1.resolvedOptions();
+
+console.log(options1.locale);
+// expected output (Firefox / Safari): "de-DE"
+// expected output (Chrome): "de"
```
Comment (wbamberg) [inline live-examples/js-examples/intl/intl-listformat-prototype-resolvedoptions.js:6]: I don't really like calling out specific browsers in here. Maybe we could avoid this issue by choosing `"fr"` instead? If so we should change the name of the variable of course.

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks! I had a few little comments.

File: live-examples/js-examples/intl/intl-listformat-prototype-resolvedoptions.js (hunk 3 above)
Comment (wbamberg) [reply live-examples/js-examples/intl/intl-listformat-prototype-resolvedoptions.js:6]: Oh I just saw that the other one uses `de`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/resolvedOptions. Well maybe keep it then, so we are consistent.

File: live-examples/js-examples/intl/intl-listformat-prototype-resolvedoptions.js (hunk 2 above)
Comment (IestynGage) [reply live-examples/js-examples/intl/intl-listformat-prototype-resolvedoptions.js]: Alright added :) plus changed the comment to long for the style console log.

File: live-examples/js-examples/intl/intl-listformat-prototype-resolvedoptions.js (hunk 3 above)
Comment (IestynGage) [reply live-examples/js-examples/intl/intl-listformat-prototype-resolvedoptions.js:6]: I'll leave the language as it is then 👍

**Comment by IestynGage** [comment]
I removed the spurious character that I accidently added when re-ordering `meta.json`, I'll also make a pull request to [mdn/content](https://github.com/mdn/content)

**Comment by wbamberg** [comment]
> I removed the spurious character that I accidently added when re-ordering `meta.json`, I'll also make a pull request to [mdn/content](https://github.com/mdn/content), when this gets merged 😄

Sounds great! Please ask if you have any q's.

**Pull request merged**